### PR TITLE
fix shutdown() for xcode 9.2

### PR DESCRIPTION
**Pull request by kool79**

### CODE REVIEW

File: lib/simctl.js [hunk 1]
```diff
@@ -315,7 +315,9 @@ async function shutdown (udid) {
   try {
     await simExec('shutdown', 0, [udid]);
   } catch (err) {
-    if ((err.message || '').indexOf('Unable to shutdown device in current state: Shutdown') === -1) {
+    let reason = err.message || '';
+    if (reason.indexOf('Unable to shutdown device in current state: Shutdown') === -1 &&
```
Comment (mykola-mokhnach) [inline lib/simctl.js]: I'd rather use 
```
if (!reason.includes('Unable to shutdown device in current state: Shutdown') || 
     !reason.includes('Unable to lookup in current state: Shutdown'))
....
```

**Comment by kool79** [comment]
fixed

**Comment by mykola-mokhnach** [comment]
Thanks. Please accept CLA, so we can merge the fix

**Comment by kool79** [comment]
Already did

**Comment by imurchie** [comment]
Thanks!

**Comment by imurchie** [comment]
Published in `node-simctl@3.11.6`.

**Comment by jlipps** [comment]
Thanks @kool79!

**Pull request merged**